docs/src/pages/agents.mdx

```diff
@@ -0,0 +1,79 @@
+## Agent Configuration Model
+
+The Agent Configuration model contains all the information and data related to an agent configuration.
+
+### Properties
```

Comment:
I omitted the action / generation config information as I don't think they are super relevant for end users at the moment (there's no POST).

Reply:
raises the question of whether we want to have separate output types for external / internal APIs. I lean towards no, but I think it makes sense to have parts of it that isn't documented

Reply:
Well for coherence we may want to have a rendering with lower_case and underscore... but we broke that rule for now.

By omitted you mean in the docs right? not in the API response?

Reply:
yeah I meant docs